---
title: "Océane Cassan's resume"
author: Océane Cassan
date: "`r Sys.Date()`"
output:
  pagedown::html_resume:
    # set it to true for a self-contained HTML page but it'll take longer to render
    self_contained: true
# uncomment this line to produce HTML and PDF in RStudio:
knit: pagedown::chrome_print
css: "style.css"
urlcolor: red
linkcolor: red
---

Aside
================================================================================


![Océane Cassan](me3.jpg){width=60%}


Contact {#contact}
--------------------------------------------------------------------------------

- <i class="fa fa-envelope"></i> oceane.cassan@lirmm.fr
- <i class="fa fa-github"></i> [github.com/OceaneCsn](https://github.com/OceaneCsn)
- <i class="fa fa-phone"></i> +33 6 79 42 48 89


Skills {#skills}
--------------------------------------------------------------------------------

- **Pluridisciplinarity** : ability to understand and articulate biology, statistics and computer science to test hypotheses

- **Statistics** : statistical learning, interpretable machine learning, (sparse) regression and feature selection, mixed models, tree-based methods

- **Computer science** : R (ggplot, tidyverse, package developement, Rmarkdown, Shiny), Python, LaTeX-beamer, Bash, calculation server administration and usage, web application deployment

- **Biology** : gene regulation, systems biology, regulatory genomics, omics data analysis (RNA-Seq, (ONT-)CAGE-Seq, TF Binding experiments, GWAS data)


Disclaimer {#disclaimer}
--------------------------------------------------------------------------------

<!-- Last updated on `r Sys.Date()`. -->



Main
================================================================================

[Océane Cassan](https://www.researchgate.net/profile/Cassan-Oceane) {#title}
--------------------------------------------------------------------------------

### Postdoctoral researcher in statistical learning for systems biology

Education {data-icon=graduation-cap data-concise=true}
--------------------------------------------------------------------------------


### University of Montpellier

PhD in Systems Biology

Montpellier, France

2022 - 2019

### University Claude Bernard (UCBL)

[MSc in Computer Science - Artificial Intelligence](http://master-info.univ-lyon1.fr/IA/)

Lyon, France

2019 - 2018

### National Institute of Applied Sciences (INSA)

[Engineering degree in Bioinformatics and Modelling](https://biosciences.insa-lyon.fr/fr/content/parcours-bioinformatique-et-modelisation)

Lyon, France 

2019 - 2014

Research {data-icon=laptop}
--------------------------------------------------------------------------------


### Postdoc - Laboratory of Computer Science, Robotics and Microelectronics of Montpellier (LIRMM)

Statistical learning to predict and interpret biological signals at transcription initiation sites from DNA sequence.

Montpellier, France

2024 - 2023

- Prediction of R-loops formation (DNA-RNA hybrids with roles in gene regulation and genomic instability) from low complexity DNA sequence features at transcription initiation sites. Collaborators : Rosemary Kiernan (IGH) and Jérôme Moreaux (CHU).
- Prediction of transcriptional activity at enhancers and promoters in the course of neuron differentiation from low complexity DNA sequence features and TF binding motifs. Collaborators : [FANTOM6](https://fantom.gsc.riken.jp/) international consortium
- Team : Joint [IGMM/LIRMM/IMAG Computational Regulatory Genomics](https://www.igmm.cnrs.fr/en/service/biogenese-micro-arns/). Supervisors: Laurent Bréhélin, Charles Lecellier

### PhD - Institute for Plant Sciences in Montpellier (IPSIM)

Systems biology approaches to discover regulatory pathways in plants under elevated CO$_2$.

Montpellier, France

2022 - 2019

- **Statistical inference of the gene regulatory networks in *Arabidopsis thaliana* under elevated CO$_2$ and nutritional limitations**. Development of novel network reconstruction methods via ensembles of regression trees or penalized linear regression based on transcriptomic data and regulatory sequences information. **Computational and experimental validation** of network-derived hypotheses and candidate genes.
- **Genome Wide Association studies** to identify genetic determinants of mineral status repsonse under elevated CO$_2$
- **Development of an [interactive suite](https://oceanecsn.github.io/DIANE/)** for reproducible transcriptomic analyses and the inference of regulatory networks
- Supervisors : Antoine Martin, Sophie Lèbre
- Team : [SIRENE](https://www1.montpellier.inra.fr/wp-inra/bpmp/en/research/the-teams/integration-of-nutrient-signaling-pathways/)

### 6 months research internship - Laboratory of Computer Science, Robotics and Microelectronics of Montpellier (LIRMM)

Supervised statistical learning to predict chromosomic interactions in the human genome, based on CAGE data and chromatin contact data.

Montpellier, France

2019

- Supervisor : Laurent Bréhélin
- Team : Joint [IGMM/LIRMM/IMAG Computational Regulatory Genomics](https://www.igmm.cnrs.fr/en/service/biogenese-micro-arns/)

### 1 month research internship - Laboratory of Computer Science, Robotics and Microelectronics of Montpellier (LIRMM)

Sparse logistic regression to model transcription factor combinatorics for gene regulation in human

Montpellier, France

2017

- Supervisor : Laurent Bréhélin
- Team : Joint [IGMM/LIRMM/IMAG Computational Regulatory Genomics](https://www.igmm.cnrs.fr/en/service/biogenese-micro-arns/)

Professional Experience {data-icon=suitcase}
--------------------------------------------------------------------------------

### 4 months internship in quantitative biology - [Nanolive](https://www.nanolive.ch/) SA

Development of software tools for the 3D detection and tracking of biological objects in label free images

Lausanne, Switzerland

2018

Teaching Experience {data-icon=chalkboard-teacher}
--------------------------------------------------------------------------------

### Instructor of R for $3^{rd}$ year bachelor plant biology students

[Teaching material](https://scortijo.github.io/2021_L3_R/)

University of Montpellier, France

2023 - 2021

:::concise
- Good practices and scientific integrity around data analysis
- Data manipulation and visualisation in R
:::

### Instructor of Rmarkdown for Master data science students

[Teaching material](https://github.com/OceaneCsn/M1_Rmarkdown_course)

University Paul Valery of Montpellier, France

2023 - 2020

### Statistical inference of regulatory networks for Master plant biology students

[Lecture](https://oceanecsn.github.io/Network_inference_practical_session/Course_slides_HAA814V-Cassan-Lebre.pdf) and [practical session](https://oceanecsn.github.io/Network_inference_practical_session/)

University of Montpellier, France

2024 - 2022

### Student supervision

N/A

N/A

2021 - 2022

:::concise
- Master 2 intern in biostatistics appied to gene regulation and transcriptomics
- Master 1 data science students for an annual project on Genome-Wide-Association studies
:::

Publications {data-icon=file}
--------------------------------------------------------------------------------

### Optimizing data integration improves Gene Regulatory Network inference in *Arabidopsis thaliana*

<span style="text-decoration:underline">Océane Cassan</span>, Charles-Henri Lecellier, Antoine Martin, Laurent Bréhélin, Sophie Lèbre. 2023, Preprint. [See preprint](https://www.biorxiv.org/content/10.1101/2023.09.29.558791v1)

N/A

2023

### A gene regulatory network reveals the effects of elevated CO$_2$ on nutrient signaling pathways and mineral composition in Arabidopsis

<span style="text-decoration:underline">Océane Cassan</span>, Léa-Lou Pimparé, Antoine Beckers, Alain Gojon, Liên Bach, Christian Dubos, Sophie Lèbre, Antoine Martin. 2022, New Phytologist. [See article](https://nph.onlinelibrary.wiley.com/doi/epdf/10.1111/nph.18788)

N/A

2023

### Natural genetic variation underlying the negative effect of elevated CO2 on ionome composition in *Arabidopsis thaliana*

<span style="text-decoration:underline">Océane Cassan</span>, Léa-Lou Pimparé, Timothy Mozzanino, Cécile Fizames, Sébastien Devidal, Fabrice Roux, Alexandru Milcu, Sophie Lèbre, Alain Gojon, Antoine Martin. 2023, Preprint. [See preprint](https://www.biorxiv.org/content/10.1101/2023.06.05.543678v1)

N/A

2023

### The decline of plant mineral nutrition under rising CO$_2$: physiological and molecular aspects of a bad deal

Alain Gojon, <span style="text-decoration:underline">Océane Cassan</span>, Liên Bach, Laurence Lejay, Antoine Martin. 2022 Review article, Trends in Plant Science. [See article](https://www.cell.com/trends/plant-science/fulltext/S1360-1385(22)00247-3)

N/A

2022

### Loss of Polycomb proteins CLF and LHP1 leads to excessive RNA degradation in Arabidopsis

David Séré, <span style="text-decoration:underline">Océane Cassan</span>, Fanny Bellegarde, Cécile Fizames, Jossia Boucherez, Geoffrey Schivre, Jacinthe Azevedo, Thierry Lagrange, Alain Gojon, Antoine Martin. 2022 Journal of Experimental Botany. [See article](https://doi.org/10.1093/jxb/erac216)

N/A

2022

### Inferring and Analyzing Gene Regulatory Networks from Multi-Factorial Expression Data: A Complete and Interactive Suite

<span style="text-decoration:underline">Océane Cassan</span>, Sophie Lèbre, and Antoine Martin. 2021 BMC Genomics 22 (1). [See article](https://doi.org/10.1186/s12864-021-07659-2)

N/A

2021


### Probing Transcription Factor Combinatorics in Different Promoter Classes and in Enhancers

Vandel Jimmy, <span style="text-decoration:underline">Océane Cassan</span>, Sophie Lèbre, Charles-Henri Lecellier, and Laurent
Bréhélin. 2019 BMC Genomics 20 (1): 1–19. [See article](https://doi.org/10.1186/s12864-018-5408-0)

N/A

2019

Conferences and seminars {data-icon=staff}
--------------------------------------------------------------------------------

### [FANTOM6 consortium meeting](https://fantom.gsc.riken.jp/6/)

Speaker

Milan, Italy

2023

- Probing the contribution of DNA low complexity regions in transcriptional regulations during neuron differentiation

### [ISMB-ECCB](https://www.iscb.org/ismbeccb2023)

Speaker

Lyon, France

2023

- Gene-specific optimisation of data integration in regression-based Gene Regulatory Network inference. **Awarded the best oral presentation in the NETBIO session.**

### [Machine Learning in Montpellier, Theory & Practice](https://groupes.renater.fr/wiki/ml-mtp/index)

Speaker

University of Montpellier, France

2022

- Discussing : Reconciling modern machine-learning practice and the classical bias–variance trade-off, Belkin et al, 2019.

### [Molecular responses of plants facing climate change](https://meetings.embo.org/event/22-plants-climate-change)

**Organizer** and poster presenter

Montpellier, France

2022

- Genetic determinants of the mineral status response under elevated CO$_2$ in Arabidopsis populations


### [JOBIM](https://jobim2021.sciencesconf.org/)

Poster presenter

Virtual

2021

- Dashboard for the Inference and Analysis of Networks from Expression data

### [International Plant Systems Biology](https://meetings.embo.org/event/20-plant-systems)

Poster presenter

Virtual

2021

- Gene Regulatory Network inference of Arabidopsis under elevated CO$_2$ and nutritional limitations

### [Netbio, biological networks inference](https://miat.inrae.fr/site/NETBIO)

Speaker

Virtual

2020

- Presenting the Dashboard for the Inference and Analysis of Networks from Expression data

### [Methods and Algorithms for Bioinformatics Seminars](https://www.lirmm.fr/teams-en/mab-en/)

Organizer

Montpellier, France

2024 - 2023

- Co-planner (with Eric Rivals) of regular team seminars on the topic of bioinformatics


Academic referees {}
--------------------------------------------------------------------------------

PhD:

#### **Antoine Martin**, senior researcher in plant biology, head of SIRENE team, IPSiM, CNRS : antoine.martin@cnrs.fr

#### **Sophie Lèbre**, lecturer and researcher in statistics, IMAG, University of Montpellier, University Paul Valéry : sophie.lebre@umontpellier.fr

Postdoc:

#### **Laurent Bréhélin**, senior researcher in computational biology, head of MAB team, LIRMM, co-head of the Computational Regulatory Genomics team : brehelin@lirmm.fr

#### **Charles Lecellier**, senior researcher in computational biology, co-head of the Computational Regulatory Genomics team : charles.lecellier@igmm.cnrs.fr



